data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml

```diff
@@ -94,65 +94,61 @@
   </langUsage>
  </profileDesc>
  <revisionDesc>
-  <change who="GRC" when="2023-05-24">GRC added 169 corrections, each marked with a <corr/> tag.</change>
```

Comment:
I would keep this note as the changes are extensive enough to warrant it. Not jus the tag removal but also the xml:base addition

Reply:
in the case where stuff was removed or added automatically, this is even more important to alert future editors, I think